scipy/sparse/construct.py

```diff
@@ -791,5 +791,20 @@ def rand(m, n, density=0.01, format="coo", dtype=None, random_state=None):
     -----
     Only float types are supported for now.
 
+    See Also
+    --------
+    scipy.sparse.random : Equivalent function when passing `data_rvs=None`
```

Comment:
The reason `rand` exists is for backwards compatibility, as it existed before `random`: https://github.com/scipy/scipy/pull/4679#issuecomment-88635022

It might be better to describe `rand` in terms of `random`, which would help point users to the more powerful function when/if they need it.